data/tokens.json

```diff
@@ -1815,6 +1819,21 @@
     "gtmds_avatar-group_border-color": {
       "value": "{gtmds_color_white}",
       "type": "color"
+    },
+    "XXLarge": {
+      "gtmds_avatar-label": {
+        "value": {
+          "fontFamily": "{gtmds_font-family-h2}",
+          "fontWeight": "{gtmds_font-weight-700}",
+          "lineHeight": "{gtmds_line-height-base}",
+          "fontSize": "{gtmds_avatar-label_font-size-xl}",
```

Comment:
Shouldn't this be an xxl?

Reply:
hmm, this is an issue, I've made it xl {gtmds_avatar-label_font-size-xl}*2 instead of xxl. I will revisit and that